### PR TITLE
Filled in the AnonCreds conventions section

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by mikelodder7 — COMMENTED** [review]
Just a few minor grammar changes nothing major.

File: spec/anoncreds_conventions.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
 ## AnonCreds Conventions
 
-::: todo
-Cover conventions like encoding claims, date handling for predicates and revocation status requests
-:::
+Over the years of using Hyperledger AnonCreds, several conventions (listed here) have been defined can or must be used
+in deploying AnonCreds. They conventions are not required in the current [AnonCreds implementation], but
```
[the remark this answers is not in the document]
Comment (swcurran) [reply spec/anoncreds_conventions.md:4]: Fixed.

File: spec/anoncreds_conventions.md [hunk 2]
```diff
@@ -1,5 +1,69 @@
 ## AnonCreds Conventions
 
-::: todo
-Cover conventions like encoding claims, date handling for predicates and revocation status requests
-:::
+Over the years of using Hyperledger AnonCreds, several conventions (listed here) have been defined can or must be used
+in deploying AnonCreds. They conventions are not required in the current [AnonCreds implementation], but 
+are necessary to be followed to be interoperable with other AnonCreds implementations.
```
[the remark this answers is not in the document]
Comment (swcurran) [reply spec/anoncreds_conventions.md:5]: Much better :-) Fixed.

**Comment by swcurran** [comment]
Thanks, @mikelodder7 — ready for approval.

**Pull request merged**